### DISCUSSION TITLE
Making a Custom Menu the default for all new pc users

### DISCUSSION THREAD

**Discussion by stevesousa1** [Open-Shell/Open-Shell-Menu#1953] [General]
"I've created shortcuts to our business folder on win7 start menu in the win admin user account . However, when a new user logs into the computer (we are on a domain) and the program starts, they usually just click 'OK' and just end up with the default Windows 7 setup, which is okay. Unfortunately, they don't get the custom win7 menu (with custome shortcuts) that I set up. Do you have any ideas on how we can create a custom Windows 7 start menu and set it as the "default" for any new users?"

**Comment by ge0rdi** [comment]
`Open-Shell Menu Settings` changes settings just for _current_ user.

If you want to manage settings for different users, see `Administrative Settings` chapter in `Open-Shell` help.

**Comment by stevesousa1** [comment]
Thank you for your help.

Unfortunately after following the help instructions I am not able to find anything that matches the help instructions, or solve the problem of locking a default custom menu for all users.  If you have any other suggestions I would greatly appreciated it.

Thanks in advance for your help

Instructions I have found are below
![image](https://github.com/user-attachments/assets/7a0393ad-d8ac-430c-845f-74417ae8b044)

**Reply by ge0rdi** [reply]
Set up the menu you want to have and then copy respective settings from `HKEY_CURRENT_USER\SOFTWARE\OpenShell\StartMenu\Settings` to `HKEY_LOCAL_MACHINE\SOFTWARE\OpenShell\StartMenu` key.

Menu style is in `MenuStyle` registry value, custom menu items in `MenuItems7`.

Alternatively, the help mentions how to install group policies. You can then edit them in `gpedit` just like any other policies (its a bit easier than dealing with registry).

**Comment by stevesousa1** [comment]
Perfect thank you so much.  I ended up making a batch program to import the registry key into local machine and now will use it for all other machines.

Thank you your lifesaver


________________________________
